app/assets/javascripts/member-facing.js.erb

```diff
@@ -19,6 +19,8 @@
 require("<%= Settings.external_js_path %>");
 <% end %>
 
+window.URI = require('urijs');
```

Comment:
I'm not crazy about importing a JS library just to add a URL param... it's pretty easy to write a function that does it. I feel like this leaves us more open to the leftpad.js problem.